docs/reference/technology/storage.md

```diff
@@ -39,7 +39,7 @@ An example of implementations of the interfaces are:
 - FileHandle.
    - File - Class that provides read and write access to files on a file system.
    - BlockDeviceFile - Class that provides read and write access to raw block device storage in the form of a FileHandle.
-   - UARTSerial - The UARTSerial class provides read and write access to a device's serial hardware. It is an example of a nonstorage device that you can access as a file.
+   - UARTSerial - The UARTSerial class provides read and write access to a device's serial hardware. It is an example of a non-storage device that you can access as a file.
```

Comment:
I don't think this should be hyphenated. "Non" ends with an "n", and "storage" doesn't start with an "n". There's no proper noun or acronym there, either.

Reply:
True, nonstorage just strikes me as... off.